pkg/istio/rbac/v2/provider.go

```diff
@@ -78,6 +80,23 @@ func (p *v2) ConvertAthenzModelIntoIstioRbac(athenzModel athenz.Model, serviceNa
 			if err != nil {
 				continue
 			}
+
+			// Drop the query parameters from the HTTP path in the assertions due to the difference
+			// in the RBAC Envoy permissions config created by Authentication Policy and ServiceRole/ServiceRoleBindings.
+			// Which in case of,
+			// Authentication Policy - is created with a url_path object
+			// ServiceRole/ServiceRoleBindings - is created with a header object
+			re, err := regexp.Compile(`.*\?.*`)
```

Comment:
Can you move this block after checking svc regex? This will reduce # of operations.

Reply:
Sure, makes sense

Reply:
Since this regex is the same for every iteration, can we move the compilation out of this function entirely so that it only needs to happen once? Maybe as a variable at the top of this file?